### DISCUSSION TITLE
Shifted main element

### DISCUSSION THREAD

**Discussion by IsabellUrban** [neuefische/questions#860] [Web] [answered]
Hi team, 
when I open our app in the browser (wider desktop view), the main element seems to move a few pixels to the left as it hits the max-width and the striped body is visible in the background (see screenshot). 

Strangely, this does not happen when I have the Dev Tools open. It also only happens on my computer – everyone else does not have this problem. 

Does anyone have any idea what could be causing this? 

Thanks in advance.

Here the [Link to vercel](https://capstone-flip-wise-git-design-elisabeths-projects-86003ae8.vercel.app/) and [to our code](https://github.com/eneben/capstone-flip-wise/tree/design)

![screenshot shifted main element](https://github.com/user-attachments/assets/eb0c43c6-982a-456b-99b3-c8c0e20eccbe)

**Comment by eneben** [comment, marked as answer]
We found the solution using Chat GPT. We set `overflow: hidden` on the `body`-element. Chat GPT explained that for some systems, scrollbars may behave differently when Dev Tools are open. When Dev Tools are closed, the scrollbar appears, shifting the layout slightly. `overflow: hidden` solved the problem.
